app/src/main/res/layout/widget_button_configure.xml

```diff
@@ -117,6 +117,28 @@
             android:orientation="horizontal">
 
             <TextView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_gravity="center_vertical"
+                android:padding="5dp"
+                android:text="@string/label_icon_color" />
+
+            <androidx.appcompat.widget.AppCompatEditText
+                android:id="@+id/icon_color"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:hint="@string/widget_text_hint_icon_color"
+                android:imeOptions="actionDone"
```

Comment:
This should not be set, as this isn't the last text field (so the keyboard action should continue to the next field).